apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -97,7 +98,7 @@ const BookingPage = ({
         collectPageParameters("/book", { isTeamBooking: document.URL.includes("team/") })
       )
     );
-  }, []);
+  }, [telemetry]);
```

Comment:
Does this achieve the desired behavior? Do we want to track only on mount or everything the telemetry object changes?

Reply:
I'm guessing the first one, the telemetry object is not expected to change. I can just ignore this instance.